test/corpus/functions.txt

```diff
@@ -16,7 +16,6 @@ selector as infix binary operators
             (number
               (integer ))))
         operator: (method_name )
-        operator: (method_name )
```

Comment:
It seems like these lines were duplicated accidentally?

Reply:
nice catch!